operator/src/main/resources/scripts/introspectDomain.sh

```diff
@@ -119,19 +119,23 @@ if [ ${DOMAIN_SOURCE_TYPE} == "FromModel" ]; then
     trace "Beginning Model In Image"
     command -v gzip
     if [ $? -ne 0 ] ; then
-      trace SEVERE "gzip is missing - image must have gzip installed " && exit 1
+      trace SEVERE "DomainSourceType is FromModel, gzip is missing in the image. Please use an image with gzip installed" && exit 1
     fi
     command -v tar
     if [ $? -ne 0 ] ; then
-      trace SEVERE "tar is missing - image must have tar installed " && exit 1
+      trace SEVERE "DomainSourceType is FromModel, tar is missing in the image. Please use an image with tar installed" && exit 1
+    fi
+    command -v unzip
+    if [ $? -ne 0 ] ; then
+      trace SEVERE "DomainSourceType is FromModel, unzip is missing in the image. Please use an image with unzip installed" && exit 1
     fi
     mkdir -p ${DOMAIN_HOME}
     if [ $? -ne 0 ] ; then
-      trace SEVERE "cannot create domain home directory '${DOMAIN_HOME}'" && exit 1
+      trace SEVERE "DomainSourceType is FromModel, cannot create domain home directory '${DOMAIN_HOME}'" && exit 1
```

Comment:
LGTM. 

Minor nits for the SEVEREs:  I think it helps to put '' around each keywords like 'tar', 'gzip', 'unzip', and 'FromModel', and end sentences with a period '.'.

Reply:
Let's make these edits and then I'll commit.

Reply:
done

Reply:
Minor nits still buzzing around:  Still missing periods at end of each SEVERE sentence.  Would be nice to have quotes around FromModel too.